discohorts/pipeline.py

```diff
@@ -97,13 +102,15 @@ def get_patient_to_work_dir(patients, work_dirs):
                         check_call(command)
```

Comment:
Re `check_call`: I'm sure there is better error handling that we can add here. How about I file an issue for that?